Lib/selectors.py

```diff
@@ -349,15 +349,16 @@ def __init__(self):
         super().__init__()
         self._selector = self._selector_cls()
 
-    def register(self, fileobj, events, data=None):
+    def register(self, fileobj, events, data=None, register_flags=None):
```

Comment:
Since this is in a method literally called `register`, I'd name the new argument simply `flags`.